libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/VoiceInstructionLoader.java

```diff
@@ -30,18 +32,20 @@
   private final Cache cache;
   private final Context context;
   private MapboxSpeech.Builder mapboxSpeechBuilder = null;
+  private SkuTokenProvider skuTokenProvider;
 
-  public VoiceInstructionLoader(Context context, String accessToken, Cache cache) {
+  public VoiceInstructionLoader(Context context, String accessToken, Cache cache, SkuTokenProvider skuTokenProvider) {
```

Comment:
`VoiceInstructionLoader` is a `public` class, not sure if we should add the `SkuTokenProvider` in the constructor and force clients to pass that in, especially because SKU logic is internal 🤔 We need to revisit this.

Reply:
I agree that this needs to be revisited